app/views/renewal/confirmations/show.html.erb

```diff
@@ -7,8 +7,8 @@
     <div class="verification-instructions">
       <p>Thank you for renewing your Chicago Tool Library membership.</p>
 
-      <% if @amount %>
-        <p><strong>Your payment of <%= @amount.format %> has been processed and applied to your account.</strong></p>
+      <% if @payment %>
+        <p><strong>Your payment has been processed and applied to your account.</strong></p>
```

Comment:
Should this be something like "Your payment is being processed and applied to your account." in case the job fails?

Reply:
Oh yes, that is a lot better! It also handles the fact that there might be a small delay which could happen for a variety of reasons.